src/connector/src/sink/deltalake.rs

```diff
@@ -98,18 +101,29 @@ impl DeltaLakeCommon {
                 deltalake::open_table_with_storage_options(s3_path.clone(), storage_options).await?
             }
             DeltaTableUrl::Local(local_path) => deltalake::open_table(local_path).await?,
+            DeltaTableUrl::Gcs(gcs_path) => {
+                let mut storage_options = HashMap::new();
+                storage_options.insert(
+                    GCS_SERVICE_ACCOUNT.to_string(),
+                    self.gcs_service_account.clone().unwrap(),
```

Comment:
If the `gcs_service_account` is not specified, the CN will panic? 🤔

If so, sounds unacceptable to me. Please do validation when `create sink`

Reply:
fixed

Reply:
The fix is inside `create_deltalake_client()`, you are sure that it will be called during `create sink`, right?